cmd/root.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/spec-tacles/spectacles.go/broker"
+	"github.com/spec-tacles/spectacles.go/gateway"
+	"github.com/spec-tacles/spectacles.go/rest"
+	"github.com/spec-tacles/spectacles.go/types"
+	"github.com/spf13/cobra"
```

Comment:
I tend to prefer my external imports to be below stdlib imports. This ensures consistency in the repo.